src/App.jsx

```diff
@@ -14,45 +16,102 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_APHABETICALY = 'alphabetical';
+const SORT_BY_LENGTH = 'length';
+
+function sortGoods(goods, sortMethod, isReversed) {
+  let sortedGoods = [...goods];
+
+  switch (sortMethod) {
+    case SORT_APHABETICALY:
+      sortedGoods = sortedGoods.sort((a, b) => a.localeCompare(b));
+      break;
+
+    case SORT_BY_LENGTH:
+      sortedGoods = sortedGoods.sort((a, b) => a.length - b.length);
+      break;
+
+    default:
+      break;
```

Comment:
The `default` value must not be empty. 
Either you need to set the default behavior or throw an error like "Passed sort type is incorrect".